packages/spread/operation/determine_claim_relevance.py

```diff
@@ -39,8 +39,9 @@ async def act_on_engaged_node(self):
 def query_claim_without_relevance_or_terminus(graph, operation_name):
     response = graph.execute_query(
         """
-        MATCH (claim:Claim)
-        WHERE NOT (claim)-[:TERMINATES]->(:Terminus)
+        MATCH (claim:Claim), (nutrient:Nutrient)
+        WHERE NOT (nutrient)-[:STOPPED_BY]->(:Stop)
```

Comment:
I think we need a WHERE elementId(nutrient) = claim.nutrient_id here to only consider the claim's nutrient

Reply:
Yeah, as is, if any nutrient exists in the graph that isn't stopped, then this continues to run. Thanks for the fix!